dashboard/checkit/wrapper/__init__.py

```diff
@@ -18,11 +18,12 @@ def sage(outcome,output_path,preview=True,images=False):
             with working_directory(outcome.bank.abspath()):
                 cmds = [
                     "sage",
-                    os.path.join(tmpdir,"wrapper.sage"),
-                    outcome.generator_path(),
-                    output_path,
-                    preview_s,
+                    os.path.join(tmpdir,"wrapper.sage"), #0
+                    outcome.generator_path(), #1
+                    output_path, #2
+                    preview_s, #3
+                    str(amount), #NEW 4
```

Comment:
I like the comments helping with arg indices, but we can remove "NEW" hre and on line 28

Reply:
I elaborated on the comment on line 21 so it's clearer these are in fact arg indices